### ISSUE TITLE
!id_map.has(p_rid.get_data()) error shows when randomly reparenting/deleting "CSGBox", "InterpolatedCamera", "OmniLight"

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#47356]
**Godot version:**
3.3.rc.custom_build. 7696f8854

**OS/device including version:**
Ubuntu 20.04

**Issue description:**
When randomly reparenting, deleting and adding "CSGBox", "InterpolatedCamera", "OmniLight" nodes, then this error is shown:
```
ERROR: getornull: Condition "!id_map.has(p_rid.get_data())" is true. Returned: __null
   At: ./core/rid.h:160.
```

Looks that this is not related to BVH

**Steps to reproduce:**
1. Run minimal project and wait

**Minimal reproduction project:**
[RegressionTestProject.zip](https://github.com/godotengine/godot/files/6204199/RegressionTestProject.zip)

**Comment by tinmanjuggernaut** [comment]
The cause is found in #53374 and the fix is coming in #54650.

**Comment by akien-mga** [comment]
Fixed by #54650.
